google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -117,6 +118,21 @@
   ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference[] documentReferences, @Nullable FieldMask fieldMask);
 
+  /**
+   * Retrieves multiple documents from Firestore while optionally applying a field mask to reduce
+   * the amount of data transmitted. Returned documents will be out of order and missing documents
+   * will not be returned.
+   *
+   * @param documentReferences Array with Document References to fetch.
+   * @param fieldMask If set, specifies the subset of fields to return.
+   * @param responseObserver ApiStreamObserver of DocumentSnapshot
+   */
+  void getAll(
+      @Nonnull DocumentReference[] documentReferences,
+      @Nullable FieldMask fieldMask,
+      final ApiStreamObserver<DocumentSnapshot> responseObserver)
+      throws Exception;
```

Comment:
Do we need to declare `throw Exception`? compare to the existing `getAll()` Please remove this declaration.

Reply:
done